### PR TITLE
Update the User Guide

### LINKED ISSUE #5: Update User Guide for v1.1
Task 3 of [week 7 TP](https://nus-cs2103-ay2021s2.github.io/website/schedule/week7/project.html).

### PR DESCRIPTION
**Pull request by Jellybeano**
Completed updating the User Guide, but command descriptions are mostly skeletal for now.

Can update as we go along!

edit: fixed typos in a new commit~

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -45,148 +53,91 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person’s tags by typing `t/` without
-    specifying any tags after it.
-
-Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
-
-### Locating persons by name: `find`
+### Ending the session: `end`
```
Comment (Yanneko) [inline docs/UserGuide.md:94]: is it just me or is this repeated

File: docs/UserGuide.md [hunk 2]
```diff
@@ -45,148 +53,91 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
-Format: `find KEYWORD [MORE_KEYWORDS]`
+Format: `end`
 
-* The search is case-insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+### Ending the session: `end`
```
Comment (Yanneko) [inline docs/UserGuide.md]: here

**Review by Yanneko — COMMENTED** [review]
'end' command explanation repeated in user guide, otherwise lgtm

File: docs/UserGuide.md [hunk 3]
```diff
@@ -3,34 +3,42 @@ layout: page
...
 
-1. Download the latest `addressbook.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
+1. Download the latest `weeblingo.jar` from [here](https/linkhere-tbc).
 
-1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
+1. Copy the file to the folder you want to use as the _home folder_ for your WeebLingo application.
 
-1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
-   ![Ui](images/Ui.png)
+1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. <br>
```
Comment (xinweit) [inline docs/UserGuide.md]: The GUI similar to the **picture** below

**Review by xinweit — COMMENTED** [review]
small typo. lgtm otherwise

**Review by Cheng20010201 — CHANGES_REQUESTED** [review]
As mentioned by @Yanneko and @xinweit, thanks!

File: docs/UserGuide.md (hunk 3 above)
Comment (Jellybeano) [reply docs/UserGuide.md]: Fixed!

File: docs/UserGuide.md (hunk 1 above)
Comment (Jellybeano) [reply docs/UserGuide.md:94]: Oh thanks for catching that haha

File: docs/UserGuide.md (hunk 2 above)
Comment (Jellybeano) [reply docs/UserGuide.md]: gots it

**Review by Cheng20010201 — APPROVED** [review]

**Pull request merged**